### PR TITLE
[INLONG-4240][Manager] Support postgres source and sink

### LINKED ISSUE #4240: [Feature][Manager] Add postgres source and sink node configuration utils
### Description

Add postgres source and sink node configuration utils

### Use case

Add postgres source and sink node configuration utils

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by baomingyu**
Add postgres source and sink node configuration utils

Fixes #4240

### CODE REVIEW

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/sink/PostgresSink.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    private String dbName;
+
+    @ApiModelProperty("Authentication for postgres")
+    private DefaultAuthentication authentication;
+
+    @ApiModelProperty("Target table name")
+    private String tableName;
+
+    @ApiModelProperty("Data format type for stream sink")
+    private DataFormat dataFormat;
```
Comment (gong) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/sink/PostgresSink.java:59]: `dataFormat` should not be need. `kafka` and `pulsar` may be need it.

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/source/PostgresSource.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    @ApiModelProperty("schema info")
+    private String schema;
+
+    @ApiModelProperty("Data table name list")
+    private List<String> tableNameList;
+
+    @ApiModelProperty("decoding pulgin name")
+    private String decodingPluginName;
+
+    @ApiModelProperty("Primary key, needed when data format is csv, json, avro")
```
Comment (gong) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/source/PostgresSource.java:74]: describe error.

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sink/postgres/PostgresSinkDTO.java [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    @ApiModelProperty("User password")
+    private String password;
+
+    @ApiModelProperty("Target database name")
+    private String dbName;
+
+    @ApiModelProperty("Target table name")
+    private String tableName;
+
+    @ApiModelProperty("Primary key is required when serializationType is json, avro")
```
Comment (gong) [inline inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sink/postgres/PostgresSinkDTO.java:58]: describe error.

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/sink/PostgresSink.java (hunk 1 above)
Comment (baomingyu) [reply inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/sink/PostgresSink.java:59]: yes, dataFormat is not need here, but  this class extend StreamSink, so must add dataFormat here.

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/source/PostgresSource.java (hunk 2 above)
Comment (baomingyu) [reply inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/source/PostgresSource.java]: done

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sink/postgres/PostgresSinkDTO.java (hunk 3 above)
Comment (baomingyu) [reply inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sink/postgres/PostgresSinkDTO.java]: done

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sort/util/LoadNodeUtils.java [hunk 4]
```diff
@@ -161,6 +165,29 @@ public static HiveLoadNode createLoadNode(HiveSinkResponse hiveSinkResponse) {
...
+                .map(sinkFieldResponse -> FieldInfoUtils.parseSinkFieldInfo(sinkFieldResponse,
+                        name))
+                .collect(Collectors.toList());
+        List<FieldRelationShip> fieldRelationShips = parseSinkFields(sinkFieldResponses, name);
+        return new PostgresLoadNode(postgresSinkResponse.getSinkName(),
+                postgresSinkResponse.getSinkName(),
+                fields, fieldRelationShips, null, null, 1,
+                null, postgresSinkResponse.getJdbcUrl(), postgresSinkResponse.getUsername(),
+                postgresSinkResponse.getPassword(),
+                postgresSinkResponse.getDbName() + "." + postgresSinkResponse.getTableName(),
```
[the remark this answers is not in the document]
Comment (baomingyu) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sort/util/LoadNodeUtils.java:187]: dbName will be more clear when configuring.

**Review by healchow — CHANGES_REQUESTED** [review]

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/sink/PostgresSink.java (hunk 1 above)
Comment (baomingyu) [reply inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/sink/PostgresSink.java:59]: done

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sort/util/ExtractNodeUtils.java [hunk 5]
```diff
@@ -249,4 +253,25 @@ public static PulsarExtractNode createExtractNode(PulsarSourceResponse pulsarSou
                 startupMode.getValue(),
                 primaryKey);
     }
+
+    /**
+     * Create PostgresExtractNode based KafkaSourceResponse
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sort/util/ExtractNodeUtils.java:258]: Not `Kafka` please.

**Review by healchow — APPROVED** [review]
LGTM

**Review by gong — APPROVED** [review]

**Pull request merged**